src/adapter/src/coord/sequencer/inner.rs

```diff
@@ -1278,9 +1278,22 @@ impl Coordinator {
         plan::DropObjectsPlan {
             drop_ids,
             object_type,
-            referenced_ids: _,
+            referenced_ids,
         }: plan::DropObjectsPlan,
     ) -> Result<ExecuteResponse, AdapterError> {
+        let referenced_ids_hashset = referenced_ids.iter().collect::<HashSet<_>>();
+        let mut objects = Vec::new();
+        for obj_id in &drop_ids {
+            if !referenced_ids_hashset.contains(&obj_id) {
+                let object_info = self.get_object_info(obj_id, session)?;
```

Comment:
I'm sorry, I totally forgot about an existing method. This can be replaced with the following:

```Rust
        ErrorMessageObjectDescription::from_id(
            obj_id.into(),
            &self.catalog().for_session(session),
        )
        .to_string();
```

Reply:
Oh that's funny -- will do 👍